lighthouse-core/gather/driver.js

```diff
@@ -310,18 +314,19 @@ class Driver {
    * @private
    * @template {keyof LH.CrdpCommands} C
    * @param {C} method
+   * @param {string|undefined} sessionId
    * @param {LH.CrdpCommands[C]['paramsType']} params
    * @return {Promise<LH.CrdpCommands[C]['returnType']>}
    */
-  _innerSendCommand(method, ...params) {
+  _innerSendCommand(method, sessionId, ...params) {
     const domainCommand = /^(\w+)\.(enable|disable)$/.exec(method);
-    if (domainCommand) {
+    if (domainCommand && !sessionId) {
```

Comment:
if we're going to expand to other targets, we should probably track enabled domains there as well...

Reply:
I'll use your point back atcha for now ;) 

It's only the two messages that are never disabled, so we can cross that bridge when we get to it IMO

Reply:
> It's only the two messages that are never disabled, so we can cross that bridge when we get to it IMO

ha, I am 👌👌 with crossing that bridge later, but maybe leave a comment about not bothering to check other targets?